### PR TITLE
Update Aragon Chat links

### PR DESCRIPTION
**Pull request by macor161**
# 🦅 Pull Request Description

- Update Aragon Chat links to Spectrum/Discord

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: ❓ Questions/Help
-about: If you have questions, please check Aragon Hack, Aragon Forum or Aragon Chat
+about: If you have questions, please check the Aragon Spectrum Community, Aragon Hack or Aragon Forum.
```
Comment (sohkai) [inline .github/ISSUE_TEMPLATE/question.md]: I wonder if we can add links for this?

File: README.md [hunk 2]
```diff
@@ -36,7 +36,11 @@
       Contributing
     </a>
     <span> | </span>
-    <a href="https://aragon.chat">
+    <a href="https://spectrum.chat/aragon">
+      Support
+    </a>
+    <span> | </span>
+    <a href="https://discordapp.com/channels/672466989217873929/684472917542830092">
```
Comment (sohkai) [inline README.md]: Let's combine support and chat in spectrum chat

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (macor161) [reply .github/ISSUE_TEMPLATE/question.md]: Not in the "about" section but links are available a little bit further down the issue template: https://github.com/aragon/aragon-cli/pull/1553/files#diff-1920b7226ceeed1523ae3615db9817f3R12

File: README.md (hunk 2 above)
Comment (macor161) [reply README.md]: Done :)

**Pull request merged**